features/FEATURE_LWIP/lwip-interface/CONTRIBUTING.md

```diff
@@ -0,0 +1,118 @@
+
+5. Create a branch for release. Use the same naming convention as previous branches.
+
+    `git branch <branch name> <tag>`
+
+    `e.g. git branch mbed-os-lwip-stable-2_0_2 STABLE-2_0_2_RELEASE_VER`
+
+5. Push the branch to the mbed OS LWIP fork repository to keep the fork in sync.
+
+    `git push origin <branch name>`
+
+    `e.g. git push origin mbed-os-lwip-stable-2_0_2`
+
+6. Fetch the previous prefixed branch from the mbed OS LWIP fork repository.
+
+    `git fetch <remote name> <remote branch>:<local branch>` 
+
+    `e.g. git fetch origin mbed-os-lwip-stable-2_0_1-prefixed:mbed-os-lwip-stable-2_0_1-prefixed`
+
+7. Checkout the latest prefixed branch.
+
+    `git checkout <prefixed branch>`
+
+    `e.g. git checkout mbed-os-lwip-stable-2_0_1-prefixed`
+
+8. Merge the new branch to the prefixed branch.
+
+    `git merge <branch name>`
+
+    `e.g. git merge mbed-os-lwip-stable-2_0_2`
+
+9. Check if there are any new or colliding files.
+
+    If there are new c-files rename those with `lwip_` prefix and make a commit about changes to the prefixed branch. Also, check that LWIP stack header file names do not collide with headers under the mbed OS directory tree. If there are collisions, rename the colliding LWIP headers with `lwip_` prefix.
+
+    If needed, rename files, and make a commit. Update the C-files to include any renamed header files.
+
+    `git mv <old filename> <new filename>`
+    
+    `git commit`    
+
+10. Rename the local prefixed branch to the new version. (Do not change branch names on remotes.)
+
+    `git branch -m <new prefixed branch>`
+
+    `e.g. git branch -m mbed-os-lwip-stable-2_0_2-prefixed`
+
+11. Push the new local prefixed branch to the mbed OS LWIP fork repository.
+
+    `git push origin <new prefixed branch>`
+
+    `e.g. git push origin mbed-os-lwip-stable-2_0_2-prefixed`
+
+### Merging the prefixed release to the mbed OS repository
+
+1. Clone the mbed OS repository.
+
+    `git clone git@github.com:ARMmbed/mbed-os.git`
+
+2. Go to the mbed OS root directory, and add the mbed OS LWIP fork repository as remote. Fetch branches from fork.
+
+    `git remote add <remote name> https://github.com/ARMmbed/lwip.git`
+
+    `e.g. git remote add lwip-fork https://github.com/ARMmbed/lwip.git`
+
+    `git fetch <remote name>`
+
+    `e.g. git fetch lwip-fork`
+
+3. Do a subtree pull for LWIP prefixed branch in the mbed OS root directory.
+
+    `git subtree pull --squash -P features/FEATURE_LWIP/lwip-interface/lwip <remote name> <new prefixed branch> -m "<commit message>"`
+
+    `e.g. git subtree pull --squash -P features/FEATURE_LWIP/lwip-interface/lwip lwip-fork mbed-os-lwip-stable-2_0_2-prefixed -m "Merged lwip 2.0.2 stable"`
+
+    After this step, there is a new commit visible in the mbed OS master branch that contains the changes.
+
+4. Verify that changes in the new commit are correct and create a new branch. Push the branch to your mbed OS fork for GitHub review.
```

Comment:
Please add a comma after "correct" because two independent clauses connected by a coordinating conjunction require a comma after the first independent clause.